apis/Google.Cloud.Vision.V1P2Beta1/Google.Cloud.Vision.V1P2Beta1.IntegrationTests/ImageAnnotatorClientTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Update all copyright to 2018, and use the new copyright statement?

Reply:
Whoops, yes, will do. I'll add that as a separate commit.